exporters/zipkin/zipkin_test.go

```diff
@@ -364,3 +366,20 @@ func TestErrorOnExportShutdownExporter(t *testing.T) {
 	assert.NoError(t, exp.Shutdown(context.Background()))
 	assert.NoError(t, exp.ExportSpans(context.Background(), nil))
 }
+
+func TestLogrFormatting(t *testing.T) {
+	format := "string %q, int %d"
+	args := []interface{}{"s", 1}
+
+	var buf bytes.Buffer
+	l := funcr.New(func(prefix, args string) {
+		_, _ = buf.WriteString(fmt.Sprint(prefix, args))
+	}, funcr.Options{})
+	exp, err := New("", WithLogr(l))
+	require.NoError(t, err)
+	exp.logf(format, args...)
+
+	want := "\"level\"=0 \"msg\"=\"string \\\"s\\\", int 1\""
```

Comment:
Lets make it easier to read 😉 

```suggestion
	want := `"level"=0 "msg"="string \"s\", int 1"`
```

Reply:
These should be equivalent. I think we can stick with the original.